modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
```

Comment:
Lets not add `add more`
Using this file you can define annotations, rewrite rules, and the ingressClassName.

As we don't support anything more at this phase, just don't want to give different expectations.

Reply:
Agree w/ Rohit. We can currently scope it to be limited custom changes that a customer can do to ingress configuration.

Reply:
Changed.